python/native_functions.py

```diff
@@ -152,6 +152,42 @@ def char_with_replace(num):
     except ValueError:
         return u'\ufffd'
 
+
+def round_without_error(number):
+    try: 
+        return round(number)
+    except:
+        return 0
+
+
+def floor_without_error(number):
+    try: 
+        return math.floor(number)
+    except:
+        return 0
+
+
+def ceil_without_error(number):
+    try: 
+        return math.ceil(number)
+    except:
+        return 0
+
+
+def range_without_error(start, end, step): 
+    try:
+        return list(range(start, end, step))
+    except:
+        return []
+
+def trim(string):
+    whitespace = [u'\u0009', u'\u000a', u'\u000a', u'\u000b', u'\u000c', u'\u0020', u'\u00a0', u'\u1680', u'\u2000', u'\u2001', u'\u2002', u'\u2003', u'\u2004', u'\u2005', u'\u2006', u'\u2007', u'\u2008', u'\u2009', u'\u200a', u'\u2028', u'\u2029', u'\u202f', u'\u205f', u'\u3000', u'\ufeff']
+    out = string
+    for character in whitespace:
+        out = out.strip(character)
+
+    return out
```

Comment:
I think this will fail for a string like `" \thello\t "`

Btw, [`str.strip`](https://docs.python.org/3/library/stdtypes.html#str.strip) can take a string of multiple characters

Reply:
Updated